django_socio_grpc/log.py

```diff
@@ -1,175 +0,0 @@
-"""
-logging utils
-"""
-import asyncio
```

Comment:
deleting this file will create breaking change and crash to app still using it.
Are we sure we want that @legau ?

Reply:
good time to do it imo

Reply:
I don't understand why removing set_log_record_factory function as it add valudable context if needed in classique log. Come to see you to talk about that